### PR TITLE
Basic gui

### PR DESCRIPTION
**Pull request by TheLocust3**
This is a basic GUI. It does not look very good yet but that is for next Sprint.

### CODE REVIEW

**Comment by Math-ias** [comment]
I was assigned, so am I tasked with review?

**Comment by simon-andrews** [comment]
Needs Javadoc comments.

**Comment by simon-andrews** [comment]
What? Mathias come back!

**Comment by Math-ias** [comment]
Can you do this? I am a bit lacking in time ATM.

Also I thought there were multiple assignments but it looks like you can only do one.

**Comment by simon-andrews** [comment]
I thought so too. Alright then.

File: src/org/robockets/runqueue/client/views/MainView.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	
+	private JFrame setupJFrame () {
+        try { // Nimbus theme
+            for (UIManager.LookAndFeelInfo info : UIManager.getInstalledLookAndFeels()) {
+                if ("Nimbus".equals(info.getName())) {
+                    UIManager.setLookAndFeel(info.getClassName());
+                    break;
+                }
+            }
+        } catch (Exception e) {}
```
Comment (simon-andrews) [inline src/org/robockets/runqueue/client/views/MainView.java]: Exceptions shouldn't pass silently.

File: src/org/robockets/runqueue/client/views/MainView.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import java.awt.Insets;
+import java.awt.event.ActionListener;
+
+import javax.swing.*;
+
+import org.robockets.runqueue.client.MainWindowListener;
+import org.robockets.runqueue.client.models.QueueModel;
+
+public class MainView {
+	private final int WIDTH = 500, HEIGHT = 280;
```
Comment (simon-andrews) [inline src/org/robockets/runqueue/client/views/MainView.java]: Height didn't seem to work on my machine. The window just opened at the maximum height possible. Can others run this software and see if this happens on their machines as well?

File: src/org/robockets/runqueue/client/views/MainView.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+import java.awt.Dimension;
+import java.awt.Font;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.GridLayout;
+import java.awt.Insets;
+import java.awt.event.ActionListener;
+
+import javax.swing.*;
```
Comment (simon-andrews) [inline src/org/robockets/runqueue/client/views/MainView.java]: Star imports are bad

**Comment by simon-andrews** [comment]
Overall looks good. Please fix the issues when you are making the interface prettier next week (especially the Javadocs, that's the only one that really matters). Merged.

**Pull request merged**